Resources/Prototypes/Entities/Structures/catwalk.yml

```diff
@@ -51,3 +51,32 @@
             max: 1
       - !type:DoActsBehavior
         acts: [ "Destruction" ]
+
+- type: entity
+  id: AcceleratingCatwalk
+  name: accelerating catwalk
+  description: High-tech catwalk that speeds up everyone who moves on it
+  parent: Catwalk
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Construction
+    node: AcceleratingCatwalk
+  - type: Physics
+    canCollide: true
+  - type: Sprite
+    sprite: Structures/accelerating_catwalk.rsi
+    drawdepth: FloorTiles
+  - type: SpeedModifierContacts
+    walkSpeedModifier: 1.5
+    sprintSpeedModifier: 1.5
```

Comment:
1.5 is extremely high (almost as fast as a jetpack) but as long as people don't start mapping it everywhere it's probably fine.

Reply:
it is in fact mass produceable. all the necessary materials need only steel and maybe some plastic. this could probably be as low as like 1.1 or 1.2. As it stands, this is really absurdly fast.